src/github.js

```diff
@@ -121,10 +121,7 @@ async function handle(opts) {
       headers,
     });
   }
-  log[utils.logLevelForStatusCode(response.status)](`Unable to fetch ${uri} (${response.status}) from GitHub`);
-  return new Response(body, {
-    status: utils.propagateStatusCode(response.status),
-  });
+  return errorResponse(log, -response.status, `Unable to fetch ${uri} (${response.status}) from GitHub`);
```

Comment:
is the `-` in `-response.status` intentional?

Reply:
yes... I harmonized the `errorResponse`. there are 2 modes, one that takes the status as is (> 0), and one that treats it as gateway error (<0):

https://github.com/adobe/helix-content-proxy/blob/9b9b44b5f402b00a5ff3b3316490fc489efd63de/src/utils.js#L81-L83